contrib/dhcp_agent/dhcp_agent.py

```diff
@@ -16,13 +16,13 @@
 
 IS_PY3 = sys.version_info[0] == 3
 if IS_PY3:
-    from dbm import gnu as gdbm
+    from dbm import gnu as cache_db
     from urllib.request import urlopen, Request
     from urllib.parse import urlencode
     from urllib.error import HTTPError
     string_types = (str,)
 else:
-    import gdbm
+    import dbm as cache_db
```

Comment:
\[Pylint\] INFO: Imports from package dbm are not grouped \[ungrouped\-imports\]